core/src/main/java/nu/yona/server/subscriptions/service/BuddyDto.java

```diff
@@ -104,16 +104,16 @@ public UserDto getUser()
 		return user;
 	}
 
-	Buddy createBuddyEntity(Translator translator)
```

Comment:
I prefer the old method for testability. Same for `determineTempNickname`.

Reply:
For testability, we can easily set a mock instance on the static member. That way, we can keep the code clean.